src/Swaps/handlers/UniswapV3Handler.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+pragma solidity ^0.8.0;
+
+import {BaseHandler} from "./BaseHandler.sol";
+import {ISwapRouterV3} from "../vendor/ISwapRouterV3.sol";
+import {ISwapper} from "../ISwapper.sol";
+
+/// @title UniswapV3Handler
+/// @custom:security-contact security@euler.xyz
+/// @author Euler Labs (https://www.eulerlabs.com/)
+/// @notice Swap handler executing exact output trades on Uniswap V3
+abstract contract UniswapV3Handler is BaseHandler {
+    address public immutable uniSwapRouterV3;
+
+    error UniswapV3Handler_InvalidPath();
+
+    constructor(address _uniSwapRouterV3) {
+        uniSwapRouterV3 = _uniSwapRouterV3;
+    }
+
+    /// @inheritdoc ISwapper
+    function swap(SwapParams memory params) public virtual override {
+        if (params.mode == MODE_EXACT_IN) revert Swapper_UnsupportedMode();
```

Comment:
What is the reason we can't support `ISwapRouterV3.exactInput`?

Reply:
For the FE we learned it made no sense to go to the router directly ever ever, other than to execute the exact output, ideally as tiny as possible